pkg/scale/varying_data_type.go

```diff
@@ -25,6 +25,7 @@ func (vdts *VaryingDataTypeSlice) Add(values ...VaryingDataTypeValue) (err error
 		copied := vdts.VaryingDataType
 		err = copied.Set(val)
 		if err != nil {
+			err = fmt.Errorf("adding VaryingDataTypeValue: %w", err)
 			return
```

Comment:
nit I know Tim is a fan of naked return, but I'm having PTSD seeing this instead of:

```suggestion
			return fmt.Errorf("adding VaryingDataTypeValue: %w", err)
```